packages/dropdown/stories/dropdown.stories.ts

```diff
@@ -9,96 +9,52 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
-import { storiesOf } from '@storybook/polymer';
-import { boolean, select } from '@storybook/addon-knobs';
-import { action } from '@storybook/addon-actions';
-import { html } from 'lit-html';
+import { html, action, boolean } from '@open-wc/demoing-storybook';
 
 import '../';
 import { Dropdown } from '../';
 import '../../menu';
 import '../../menu-item';
+import { TemplateResult } from 'lit-html';
 
-storiesOf('Dropdown', module)
-    .add('Default', () => {
-        return html`
-            <sp-dropdown
-                ?disabled=${boolean('Is Disabled', false, 'Component')}
-                ?invalid=${boolean('Is Invalid', false, 'Component')}
-                ?quiet=${boolean('Is Quiet', false, 'Component')}
-                @change="${(event: Event) => {
-                    const dropdown = event.target as Dropdown;
-                    action(`Change: ${dropdown.value}`)();
-                }}"
-            >
-                Select a Country with a very long label, too long in fact
-                <sp-menu slot="options" role="listbox">
-                    <sp-menu-item>
-                        Deselect
-                    </sp-menu-item>
-                    <sp-menu-item>
-                        Select Inverse
-                    </sp-menu-item>
-                    <sp-menu-item>
-                        Feather...
-                    </sp-menu-item>
-                    <sp-menu-item>
-                        Select and Mask...
-                    </sp-menu-item>
-                    <sp-menu-divider></sp-menu-divider>
-                    <sp-menu-item>
-                        Save Selection
-                    </sp-menu-item>
-                    <sp-menu-item disabled>
-                        Make Work Path
-                    </sp-menu-item>
-                </sp-menu>
-            </sp-dropdown>
-        `;
-    })
-    .add('Initial Value', () => {
```

Comment:
Was this test redundant? Seems to have been dropped.

Reply:
Add this story (and a visual regress in associate with it) back to the repo. Thanks for the second set of eyes!